pkg/reconciler/kubernetes/tektonpipeline/controller.go

```diff
@@ -39,6 +40,7 @@ import (
 )
 
 const releaseLabel = "pipeline.tekton.dev/release"
```

Comment:
we can remove this now :)

Reply:
ah ok you are using both of them, but we can just skip reading from crd
as configmap was added a few releases back and now if configmap is missing then something is wrong with the release yaml

Reply:
wdyt?

Reply:
what is somebody tries to install Pipelines version 0.24.x and Triggers version 0.14.x ?

Reply:
also user has complete freedom to delete the configmap

Reply:
yeah we don't need to support older versiosn
this is possible only when user has cloned the repo and doing make apply with required version

Reply:
when we create a release we use specific versions of components

Reply:
> also user has complete freedom to delete the configmap

this is not allowed.. operator will create it again in next reconcilation 😄

Reply:
I am fine removing the code for fetching the version from CRDs. :smile: 
my main objective was to make it backward compatible